tools/devservices_healthcheck.py

```diff
@@ -8,7 +8,7 @@
 def run_cmd(
     args: list[str],
     *,
-    retries: int = 3,
+    retries: int = 6,
     timeout: int = 5,
```

Comment:
Would it be worth making this configurable by an environment variable so CI can set something longer?